### DISCUSSION TITLE
Install plugins in parallel

### DISCUSSION THREAD

**Discussion by yasharne** [grafana/grafana#46312] [Plugins]
Hi
I have deployed grafana to Kubernetes, in order to add the plugins to the grafana, I've added them to `GF_INSTALL_PLUGINS` but the problem is when the list of plugins grows, it takes a long time to download them all. something like downloading them in parallel may be helpful

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
